### DISCUSSION TITLE
Black Screen / Pixelated ?!

### DISCUSSION THREAD

**Discussion by lost-in-emotions** [lllyasviel/stable-diffusion-webui-forge#1805] [Q&A]
Hey everyone, what i am doing wrong when my output rendered image is only black, or when i play around (dont know what im doing) i get a blurry pixelated image in the direction i want to make?! thanks for help.

**Comment by wisdombox44** [comment]
Which model, Are you using? If Flux try with different VEA, I had simmillar issue. I fixed changing vea  https://huggingface.co/black-forest-labs/FLUX.1-dev/tree/main. if not work try different VEA

**Reply by lost-in-emotions** [reply]
flux1-dev-bnb-nf4-v2.safetensors
flux1-schnell-bnb-nf4.safetensors

**Reply by lost-in-emotions** [reply]
> https://huggingface.co/black-forest-labs/FLUX.1-dev/tree/main.

HOLY 24 GB?

what VAE is good to use? i dont have any after the installation...
